ccd-definition/State.json

```diff
@@ -6,5 +6,13 @@
     "Name": "Open",
```

Comment:
Should that be changed to `In progress` or `Draft`? Two states should not have same name.